digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala

```diff
@@ -189,14 +189,13 @@ class Digiroad2Api(val roadLinkService: RoadLinkService,
 
     val location = userPreferences match {
       case Some(preference) => Some(preference._1.toDouble, preference._2.toDouble, preference._3)
-      case _ if user.configuration.authorizedMunicipalities.isEmpty || user.configuration.authorizedAreas.isEmpty => None
       case _ =>
-        if(user.isMunicipalityMaintainer())
+        if(user.isMunicipalityMaintainer() && user.configuration.authorizedMunicipalities.nonEmpty )
           getStartUpParameters(user.configuration.authorizedMunicipalities, municipalityDao.getCenterViewMunicipality)
         else {
-          if (user.isServiceRoadMaintainer())
+          if (user.isServiceRoadMaintainer() && user.configuration.authorizedAreas.nonEmpty)
             getStartUpParameters(user.configuration.authorizedAreas, municipalityDao.getCenterViewArea)
-          else if (user.isBusStopMaintainer()) //case ely maintainer
+          else if (user.isBusStopMaintainer() && user.configuration.authorizedAreas.nonEmpty) //case ely maintainer
```

Comment:
Should't be "user.configuration.authorizedMunicipalities.nonEmpty" since inside the if we are using user.configuration.authorizedMunicipalities?